src/components/pages/article/article-page.page.tsx

```diff
@@ -0,0 +1,52 @@
+import { ArticleContent } from '../../atoms/article-content/article-content';
```

Comment:
this file doesnt need to be `.page.tsx` anymore

Reply:
Correct!